src/statemachine.rs

```diff
@@ -668,6 +671,44 @@ impl StateMachineCtap2 {
         }
     }
 
+    pub fn reset_helper(
```

Comment:
The problem here is: We cannot do as you propose, because of the code structure. The manage-function runs within the context of the device. If the user unplugs and replugs the device, that will trigger a new thread to run.

The way I currently do it, is the following:
1. Start up the manage-function as normal, send out the authenticator-info
2. Determine from the auth-info (or rather its existence) that the token is resettable
3. Let the user click the Reset-button, but don't send anything to auth-rs.
4. Prompt the user to unplug all devices and replug the device to be reset
5. Count the "DeviceUnavailable"-events to see when no tokens are left
6. Wait for an "DeviceAvailable"-event.
7. Send "Reset"-command and prompt user to touch the device.